libs/ui/src/examples/TanstackQuery/E3_MultiFetchQuery.tsx

```diff
@@ -4,11 +4,11 @@ import { useState } from 'react'
 
 import { Code, Select, Stack } from '@chakra-ui/react'
 import { useQuery } from '@tanstack/react-query'
-import { User } from '@wsvvrijheid/types'
-import { request } from '@wsvvrijheid/utils'
+import { Blog, StrapiModel, User } from '@wsvvrijheid/types'
+import { Request } from '@wsvvrijheid/utils'
 
 const getAuthors = () => {
-  return request<User[]>({
+  return Request.collection<StrapiModel[]>({
```

Comment:
If we know the response type we should specify it. But if the response is dynamically changes by the url, then you can use `StrapiModel`

```suggestion
  return Request.collection<User[]>({
```